api/src/main/resources/workbench-api.yaml

```diff
@@ -6345,7 +6446,7 @@ definitions:
           University Medical Center'
       duaTypeEnum:
         "$ref": "#/definitions/DuaType"
-        description: 'DEPRECATED. Replaced by registeredTierMembershipRequirement. Type of Data Use Agreement Institution has signed, 
+        description: 'DEPRECATED. Replaced by registeredTierMembershipRequirement. Type of Data Use Agreement Institution has signed,
```

Comment:
nit: unrelated - revert

Reply:
It's weird here. I don't see the empty space in my local env on master branch